resources/cdk/transcribe-streaming-unauth-role/README.md

```diff
@@ -0,0 +1,110 @@
+* ``package.json``
+* ``setup.ts``
+* ``tsconfig.json``
+
+**Important**: Update the [README.md in the resources/cdk folder of the AWS Code Examples Repository](https://github.com/awsdocs/aws-doc-sdk-examples/tree/master/resources/cdk) with a description of the resources that your AWS CDK app provisions.
+
+You can regenerate these unmerged folders and files on your development machine as needed by running the preceding ``npm`` and ``cdk`` commands on your development machine (after of course installing the AWS CDK prerequisites).
+
+Merge the **SetupStack.template.json** file into the same location(s) as any code example file(s) with which this **SetupStack.template.json** file is intended to be used.
+
+## Using the AWS CLI or the AWS CloudFormation Console
+
+Callers can use the AWS Command Line Interface (AWS CLI) or the AWS CloudFormation Console to run your AWS CloudFormation template file. (They can of course use the AWS CDK, but they would need to first install the AWS CDK prerequisites.)
+
+To use the AWS CLI, run the following command from within the same folder as the ``SetupStack.template.json`` file to create the corresponding stack in AWS CloudFormation, where ``SetupStack`` is some stack name that is unique within an individual AWS Region for the AWS account:
+
+``aws cloudformation create-stack --stack-name SetupStack --template-body file://SetupStack.template.json``
+
+The ``setup.ts`` file within this folder contains instructions for getting any custom inputs from the caller as desired and using those inputs as part of the ``aws cloudformation create-stack`` command.
+
+To delete the corresponding stack in AWS CloudFormation, run the following command:
+
+``aws cloudformation delete-stack --stack-name SetupStack``
+
+To use the AWS CloudFormation Console, do the following to create the corresponding stack:
+
+1. Sign in to the AWS CloudFormation Console, at https://console.aws.amazon.com/cloudformation
+1. Choose the desired AWS Region in which to create the corresponding stack.
+1. Choose **Create stack, With new resources (standard)**.
+1. On the **Create stack** page, for **Specify template**, choose **Upload a template file**.
+1. Choose **Choose file**.
+1. Browse to and open the ``SetupStack.template.json`` file.
+1. Choose **Next**.
+1. On the **Specify stack details** page, for **Stack name**, type a unique name for the stack. (This stack name must be unique within an individual AWS Region for the AWS account.)
+1. Specify any custom inputs, if required by the template's logic, and then choose **Next**. 
+1. On the **Configure stack options** page, choose **Next**.
+1. On the **Review** page, choose **Create stack**.
+
+To delete the corresponding stack in the AWS CloudFormation Console, do the following:
+
+1. Choose the name of the stack to delete.
+1. Choose **Delete**.
+1. Choose **Delete stack**.
+
+---
+
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+
```

Comment:
This is the instructional README meant for setting up a CDK project. Replace with specific README.